pkg/platform/packet/packet.go

```diff
@@ -94,6 +94,7 @@ type config struct {
 	DisableSelfHostedKubelet bool              `hcl:"disable_self_hosted_kubelet,optional"`
 	OIDC                     *oidc.Config      `hcl:"oidc,block"`
 	EnableTLSBootstrap       bool              `hcl:"enable_tls_bootstrap,optional"`
+	EncryptPodsTraffic       bool              `hcl:"encrypt_pods_traffic,optional"`
```

Comment:
It seems to me that this option should be available on all platforms?

Reply:
What I found was calico-host-protection is getting applied for packet only.

Reply:
@knrt10 true. We don't need Calico's host protection on platforms which provide a VPC functionality (e.g. AWS). This isn't related to encryption though.

Reply:
Yes, I have removed the felix config from calico-host-protection, and added it to calico charts, so that all encryption can be enabled for all clusters